config/manifest/eksa-components.yaml

```diff
@@ -3121,36 +3091,6 @@ spec:
                               description: URI points to the manifest yaml file
                               type: string
                           type: object
-                        syncer:
```

Comment:
Are these being removed as a side effect?

Reply:
We deprecated them in the previous PR and removed their usage from the codebase and now removing the fields that aren't being used.